shell/ci/testing/setup-devenv.sh

```diff
@@ -92,6 +96,11 @@ if [[ $PROVISION == "true" ]]; then
   exec devenv --skip-update provision $PROVISION_ARGS
 fi
 
+# If scripts/devenv/post-e2e-deploy.sh exists within the project then run it.
+if [[ -f "$PROJECT_DIR/scripts/devenv/post-e2e-deploy.sh" ]]; then
```

Comment:
Can you add documentation to the e2e docs in `docs/makefile.md`? We can get more proper in the docs in the future, but would be good to note that this exists.

Longer term we should move away from shell scripts entirely, but this makes sense with the current state of this repository